src/index.html

```diff
@@ -1,14 +1,72 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
-  </head>
-  <body>
-    <h1>Moyo header</h1>
-  </body>
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Moyo header</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./style.css">
+</head>
+
+<body>
+  <header>
+    <div class="moyo-header">
+      <a href="images/logo.png" class="image">
+        <img src="images/logo.png" alt="">
+      </a>
+      <div class="nav__box">
+        <nav>
+          <ul class="nav__list">
+            <li>
+              <a href="" class="nav__item item1 is-active">
```

Comment:
item1, item2, item3 ..... 
1. Class names should represent the meaning of the content
2. You are not providing any unique styles for all these classes
3. Try to use only **is-active** class